ui-components/src/forms/Field.tsx

```diff
@@ -69,7 +84,7 @@ const Field = (props: FieldProps) => {
       {!isRadioOrCheckbox && label}
       {note}
       <div className={controlClasses.join(" ")}>
-        {props.prepend && <span className="prepend">{props.prepend}</span>}
+        {props.type === "currency" && <span className="prepend">{`$`}</span>}
```

Comment:
hmm, maybe would be better to keep a prop for the "prepend" text to do not hardcode this value?